debian/rules

```diff
@@ -20,6 +20,8 @@ major=$(shell echo $(full_version) | cut -d'.' -f1)
 minor=$(shell echo $(full_version) | cut -d'.' -f2)
 patch=$(shell echo $(full_version) | cut -d'.' -f3)
 
+# CONFIGURE_FLAGS += -DCMAKE_BUILD_TYPE=Coverage
```

Comment:
Yes, this is a flag that we don't want to use in production, because it makes the code bigger and slower, and it would make it produce coverage stats on every run. But it's still a flag that I find useful as a developer, so having it here in the debian/rules file makes it easier for me to enable it when needed.

However, a better way would be to make the flag conditional on the `DEB_BUILD_OPTIONS` variable. I'll change this.

Reply:
Done!